tests/Doctrine/Tests/DBAL/Functional/Ticket/DBAL3516Test.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\DBAL\Functional\Ticket;
+
+use Doctrine\Tests\DbalFunctionalTestCase;
+
+/**
+ * @group DBAL-3516
+ */
+class DBAL3516Test extends DbalFunctionalTestCase
```

Comment:
Please move it under `Doctrine\Tests\DBAL\Functional\ConnectionTest`.

Reply:
Ok